### DISCUSSION TITLE
Web Gui

### DISCUSSION THREAD

**Discussion by lightface79** [Nimdy/Dedicated_Valheim_Server_Script#301] [Q&A]
Hi!!!!   long term user of the njord =) 
Saw that there is a webgui on the beta...   how do i enable it?
Just did a fresh install but i culd not get the beta archive to install..
or am i missing somthing?

Best regads janne/Swedish viking =)

**Comment by Nimdy** [comment]
i need to figure out how to make it secure... ;)
